cpp/oneapi/dal/detail/error_messages.cpp

```diff
@@ -82,6 +82,8 @@ MSG(object_does_not_provide_read_access_to_columns,
     "Given object does not provide read access to columns")
 MSG(object_does_not_provide_write_access_to_columns,
     "Given object does not provide write access to columns")
+MSG(object_does_not_provide_access_to_rows_or_columns,
```

Comment:
Place it in some "compatibility" section?

Reply:
I don't see strong reason. Other error messages are used in the compatibility functionality too. There is no obvious rule that decides whether an error message should be moved to this section.